### PR TITLE
Add team page

### LINKED ISSUE #848: Who are we (the team), what are we about, who are we to say who respects privacy and who doesn't?
> I wonder if there should be a "who are we" page explaining why we are in and what we do etc.
> I am reading #779 (Signal removal) and it brings me that question, who are we and what are we after, as I don't think I know

~ Me in the team room, I think this also ties in with https://github.com/privacytoolsIO/privacytools.io/issues/780 maybe https://github.com/privacytoolsIO/privacytools.io/issues/847 and a lot more.

Personally I am just interested in privacy and tools which with I have more privacy and I find the discussions on this issue tracker often very interesting. One day I just received an invite to the repository and another day to the organisation.

> I am also not entirely sure what I am supposed to do, but I guess continuing as before is fine as it landed me here.

~ Part of what I said in GitHub team team page. I also haven't pull requested much as I want to avoid touching the HTML directly (is it even valid HTML?) and I would prefer to edit markdown files.

I think a page answering the questions in the issue title is now even more important as there are services hosted by Privacytools.io which in my opinion isn't as reliable as knowing who is behind it and what are their motives/why are they doing this.

### PR DESCRIPTION
**Pull request by JonahAragon**
## Description

Todo:

- [x] Add team page and members
- [x] Add mission statement (#848) and details
- [x] Link page in navbar and other areas.

Closes #1447, Resolves #848.

* Netlify preview for the mainly edited page: https://deploy-preview-1473--privacytools-io.netlify.com/about/

### CODE REVIEW

File: pages/team.html [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+website="https://www.danarel.com"
+twitter="danarel"
+mastodon="https://mastodon.technology/@danarel"
+%}
+
+{% include team.html
+avatar="dawid.png"
+name="Dawid Potocki"
+role="Developer"
+bio=""
```
Comment (JonahAragon) [inline pages/team.html]: @dawidpotocki please suggest a bio here.

File: pages/team.html [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+mastodon="https://social.privacytools.io/@dawidpotocki"
+%}
+
+{% include team.html
+avatar="dngray.png"
+name="dngray"
+nick="Daniel Gray"
+bio=""
```
Comment (JonahAragon) [inline pages/team.html]: @dngray please suggest a bio here.

File: pages/team.html [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+email="dngray@privacytools.io"
+%}
+
+{% include team.html
+avatar="mikaela.png"
+name="Mikaela Suomalainen"
+nick="Mikaela"
+bio=""
```
Comment (JonahAragon) [inline pages/team.html]: @Mikaela please suggest a bio here.

File: pages/team.html [hunk 4]
```diff
@@ -0,0 +1,106 @@
...
+website="https://mikaela.info/"
+mastodon="https://mastodon.partipirate.org/@mikaela"
+%}
+
+{% include team.html
+avatar="nitrohorse.jpg"
+name="Andreas"
+nick="nitrohorse"
+role="Developer"
+bio=""
```
Comment (JonahAragon) [inline pages/team.html]: @nitrohorse please suggest a bio here.

File: pages/team.html [hunk 5]
```diff
@@ -0,0 +1,106 @@
...
+website="https://nitrohorse.com/"
+mastodon="https://nitro.horse/@andreas"
```
Comment (JonahAragon) [inline pages/team.html]:
```suggestion
mastodon="https://nitro.horse/@andreas"
blog="nitrohorse"
```

Forgot I needed to add this (links to https://blog.privacytools.io/author/nitrohorse/).

File: pages/team.html [hunk 6]
```diff
@@ -0,0 +1,106 @@
...
+%}
+
+{% include team.html
+avatar="danarel.jpg"
+name="Dan Arel"
+role="ThinkPrivacy Editor"
+bio="I'm an author, columnist, and privacy advocate whose work has appeared in Time, Huff Post, OpenSource, and more. I am editing ThinkPrivacy, a PrivacyTools resource for everyday users."
+website="https://www.danarel.com"
+twitter="danarel"
+mastodon="https://mastodon.technology/@danarel"
```
Comment (JonahAragon) [inline pages/team.html]:
```suggestion
mastodon="https://mastodon.technology/@danarel"
blog="danarel"
```

Forgot I needed to add this (links to https://blog.privacytools.io/author/danarel/)

File: pages/team.html (hunk 4 above)
Comment (nitrohorse) [reply pages/team.html]: name=“Andreas Basurto”
...
bio="I'm a privacy advocate and software developer. I write web extensions, help research, and make updates to the site.”
website="https://nitrohorse.com"
mastodon="https://nitro.horse/@andreas"
blog="nitrohorse"

File: pages/team.html [hunk 7]
```diff
@@ -0,0 +1,106 @@
...
+name="Mikaela Suomalainen"
+nick="Mikaela"
+bio=""
+website="https://mikaela.info/"
+mastodon="https://mastodon.partipirate.org/@mikaela"
+%}
+
+{% include team.html
+avatar="nitrohorse.jpg"
+name="Andreas"
```
Comment (JonahAragon) [inline pages/team.html]:
```suggestion
name="Andreas Basurto"
```

File: pages/team.html [hunk 8]
```diff
@@ -0,0 +1,108 @@
...
+name="dngray"
+nick="Daniel Gray"
+bio=""
+email="dngray@privacytools.io"
+%}
+
+{% include team.html
+avatar="mikaela.png"
+name="Mikaela Suomalainen"
+nick="Mikaela"
```
Comment (Mikaela) [inline pages/team.html]: I think Ciblia might make more sense to list here?

File: pages/about.html [hunk 9]
```diff
@@ -0,0 +1,138 @@
...
+twitter="danarel"
+mastodon="https://mastodon.technology/@danarel"
+blog="danarel"
+%}
+
+{% include team.html
+avatar="dawid.png"
+name="Dawid Potocki"
+role="Developer"
+bio="I am a web developer for this site. I protect our users eyes from the light, adding quality of life improvements like Dark Mode."
```
Comment (dawidpotocki) [inline pages/about.html]:
```suggestion
bio="I'm a student interested in software development. I help improve PrivacyTools and promote using free software."
website="https://dawidpotocki.com"
email="dawidpotocki@privacytools.io"
```

File: pages/about.html [hunk 10]
```diff
@@ -0,0 +1,140 @@
...
+twitter="danarel"
+mastodon="https://mastodon.technology/@danarel"
+blog="danarel"
+%}
+
+{% include team.html
+avatar="dawid.png"
+name="Dawid Potocki"
+role="Developer"
+bio="I'm a student interested in software development. I help improve PrivacyTools and promote using free software."
```
Comment (dawidpotocki) [inline pages/about.html]:
```suggestion
bio="I'm a student interested in software development. I help improve PrivacyTools and promote using free libre software."
```

File: _includes/team.html [hunk 11]
```diff
@@ -0,0 +1,44 @@
+<div class="col-12 col-sm-8 col-md-6 col-lg-4 col-xl-3">
+  <div class="card mb-3" itemscope itemtype="http://schema.org/Person">
```
Comment (dawidpotocki) [inline _includes/team.html]:
```suggestion
  <div class="card mb-3 mx-4 mx-sm-0" itemscope itemtype="http://schema.org/Person">
```

File: _includes/team.html [hunk 12]
```diff
@@ -0,0 +1,44 @@
+<div class="col-12 col-sm-8 col-md-6 col-lg-4 col-xl-3">
```
Comment (dawidpotocki) [inline _includes/team.html]:
```suggestion
<div class="col-12 col-sm-6 col-lg-4 col-xl-3">
```

File: pages/about.html [hunk 13]
```diff
@@ -0,0 +1,140 @@
...
+nick="Daniel Gray"
+bio="I research and communicate with privacy-focused service providers to refine our recommendations, like email and VPN providers."
+email="dngray@privacytools.io"
+%}
+
+{% include team.html
+avatar="mikaela.png"
+name="Mikaela Suomalainen"
+nick="Mikaela"
+bio="I am interested in instant messengers. I review privacy tools, and participate in our forum, matrix room, and issue tracker."
```
Comment (Mikaela) [inline pages/about.html]:
```suggestion
bio="I am interested in instant messengers. I review privacy tools, and participate in our forum, Matrix room, and issue tracker."
```

I am not sure how much this reflects me, but I guess I can PR-change it anytime, but I can surely say that names are written with a capital letter.

File: pages/about.html [hunk 14]
```diff
@@ -0,0 +1,140 @@
...
+name="dngray"
+nick="Daniel Gray"
+bio="I research and communicate with privacy-focused service providers to refine our recommendations, like email and VPN providers."
+email="dngray@privacytools.io"
+%}
+
+{% include team.html
+avatar="mikaela.png"
+name="Mikaela Suomalainen"
+nick="Mikaela"
```
Comment (Mikaela) [inline pages/about.html]: I wonder if Ciblia would make more sense?

**Pull request merged**